src/pages/sym.mdx

```diff
@@ -0,0 +1,132 @@
+Encountering one may produce a warning.
+
+Otherwise, the line defines a single **symbol**: the first token provides its location, the second token provides its name, and further tokens (if any) may provide extra metadata.
+Extra tokens are currently not specified, but may be so in the future; so, implementations should ignore them, and may produce a warning in doing so.
+
+#### Symbols
+
+A symbol is uniquely defined by the combination of its location and its name; symbol names are case-sensitive.
+If the same symbol (identical name *and* identical location) is defined twice, one of the two definitions must be silently ignored.
+
+If two distinct symbols are defined with the same name (but different locations), neither definition shall be ignored, and this should not be an error.
+A warning may be produced, though not necessarily when loading the sym file: for example, "displaying names instead of memory addresses" accomodates this case just fine, however a "go to symbol" feature wouldn't, and a warning could be produced when the user tries to go to an ambiguous target.
+
+##### Location
+
+The location shall be of the form `<bank>:<address>`, both `<bank>` and `<address>` being case-insensitive hexadecimal numbers, possibly zero-padded, separated by a colon (`:`, U+003A).
+Typical hexadecimal prefixes (`0x`, `$`, etc.) are not allowed.
+An implementation may also allow the bank to be the case-insensitive keyword `BOOT` to specify symbols in the loaded boot ROM.
+If the bank is omitted, then the colon must be as well; the results of such lines are implementation-defined.
+
+The address must be in 16-bit range (0 to $FFFF inclusive), the bank must be in 32-bit range (0 to $FFFFFFFF inclusive); implementations may abandon processing the line if either number is larger than these limits.
+Implementations may accept bank numbers higher than $FFFFFFFF.
+Nonsensical locations (e.g. `2:9800`) may produce a warning.
+
+Two locations are considered identical if they have the same bank and the same address.
+Two distinct symbols can have identical locations; in particular, <LatestStableLink to="rgbasm.5#Unions">unions</LatestStableLink> facilitate this.
+
+Locations are strictly ordered by memory region (ROM0, ROMX, VRAM, SRAM, WRAM0, WRAMX, Echo RAM, OAM, I/O, HRAM, IE), then by bank, then by address.
+Entries may or may not be sorted in the sym file; for example, RGBLINK before v0.4.0 output symbols in the same order that it read them from its input files.
+Some sym files are piped through [`sort(1)`](https://man.openbsd.org/sort.1), which often produces files sorted by bank then address, but not by region.
+
+Sym files do not carry information about the memory region a symbol belongs to, but that can be inferred from the address most of the time.
+This detection is sometimes ambiguous: for example, `0:A000` could be one past the end of VRAM, or at the beginning of SRAM.
```

Comment:
Everything from here until the end of the paragraph should migrate to a non-normative section.

Reply:
The should/may/must/etc keywords are already sufficient to tell what's required versus normative. No need to spread related information across distant sections for that.

Reply:
The rest of the information isn't even related; it's just a recommendation about how to assign symbols to memory regions, which isn't part of the specification at all! Unless this has direct relevance to the specification (in which case, it should be a must — and it's a _terrible_ idea), I'd be happy to see it deleted entirely. But since someone might find the recommendations useful, moving them to the bottom seems like a reasonable compromise.

Reply:
Recommendations far away from the thing they refer are worse, since having to jump around in the document is terrible for comprehension. This is why it's here.

These paragraphs can be made into "boxes" to make their non-normative status clearer.

Reply:
This one case in particular is significant because (1) it's talking about heuristics, (2) it's introducing a lot of novel behavior that isn't related to sym file parsing, and (3) it brings in a whole lot of problems that are neither useful nor necessary arising from depending on the textual meaning of labels (e.g., what if the symbol names aren't in English?).
It's a huge tangent that seems completely out of place and adds nothing to the spec at all.

Other smaller notes are fine like that, and sometimes they don't even need a box (a parenthetical is fine), but this one case is particularly egregious. I'd really like to see it simply deleted. But if you think it's important, it really doesn't belong in the main body of the specification.